emsdk.py

```diff
@@ -1563,14 +1565,24 @@ def generate_em_config(active_tools, permanently_activate, system):
     for p in path_add:
       print('    ' + p)
     print('- This can be done for the current shell by running:')
-    print('    source "%s"' % emsdk_env)
-    print('- Configure emsdk in your shell startup scripts by running:')
     shell = os.getenv('SHELL', '')
     if 'zsh' in shell:
+      print('    source "%s"' % emsdk_env)
+      print('- Configure emsdk in your shell startup scripts by running:')
```

Comment:
Can we avoid duplicating these two lines N types by instead setting `emsdk_env` correctly above?   Perhaps add a function called `get_emsdk_env_script()`?